docs/sources/operations/storage/wal.md

```diff
@@ -83,7 +83,7 @@ Scaling up is same as what you would do without WAL or statefulsets. Nothing to
 
 When scaling down, we must ensure existing data on the leaving ingesters are flushed to storage instead of just the WAL. This is because we won't be replaying the WAL on an ingester that will no longer exist and we need to make sure the data is not orphaned.
 
-Consider you have 4 ingesters `ingester-0 ingester-1 ingester-2 ingester-3` and you want to scale down to 2 ingesters, the ingesters which will be shutdown according to statefulset rules are `ingester-3` and then `ingester-2`.
+Consider you have 4 ingesters `ingester-0 ingester-1 ingester-2 ingester-3` and you want to scale down to 2 ingesters, the ingesters which will be shutdown according to StatefulSet rules are `ingester-3` and then `ingester-2`.
```

Comment:
nit: while we are here, we can replace the noun "shutdown" with the verb form "shut down".

```suggestion
Consider you have 4 ingesters `ingester-0 ingester-1 ingester-2 ingester-3` and you want to scale down to 2 ingesters, the ingesters which will be shut down according to StatefulSet rules are `ingester-3` and then `ingester-2`.
```